bench_test.go

```diff
@@ -1,3 +1,4 @@
+//+build !race
```

Comment:
this is a bit scary

Reply:
Hm. I actually meant to apply this to mplex, not here... (one of the benchmarks there fails if it's too slow).